x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -177,16 +176,29 @@ export class HeadlessChromiumDriverFactory {
     });
   }
 
-  getBrowserLogger(page: Page): Rx.Observable<ConsoleMessage> {
-    return Rx.fromEvent(page as NodeJS.EventEmitter, 'console').pipe(
-      tap((line: ConsoleMessage) => {
+  getBrowserLogger(page: Page): Rx.Observable<void> {
+    const consoleMessages$ = Rx.fromEvent<ConsoleMessage>(page, 'console').pipe(
+      map(line => {
         if (line.type() === 'error') {
           this.logger.error(line.text(), ['headless-browser-console']);
         } else {
           this.logger.debug(line.text(), [`headless-browser-console:${line.type()}`]);
         }
       })
     );
+
+    const pageRequestFailed$ = Rx.fromEvent<PuppeteerRequest>(page, 'requestfailed').pipe(
+      map(req => {
+        const failure = req.failure && req.failure();
+        if (failure) {
+          this.logger.warning(
+            `Request to [${req.url()}] failed! [${failure.errorText}]. This error will be ignored.`
```

Comment:
This would make page errors (such as the newsfeed service being unavailable) to be logged as a warning instead of cause the reporting job to fail.